src/main/java/dan200/computercraft/shared/computer/apis/CommandAPI.java

```diff
@@ -98,6 +99,7 @@ public void shutdown()
             }
             catch( Throwable t )
             {
+                ComputerCraft.log.error( "Error running command.", t );
```

Comment:
@dan200  surely a config option to disable debug output would fix that though, while still allowing developers access to debug info when they need it.

granted @SquidDev  has a point that logs are useful in "wild encounters" but ultimatly it's a traidoff. Could a cooldown per peripheral/computer be useful? so no one problem can spam the log?